tardis_env3.yml

```diff
@@ -14,7 +14,7 @@ dependencies:
   - astropy=3
   - numba=0.53
   - numexpr
-  - pyne=0.7=nomoab_openmc*  # Issue #1537
+  - radioactivedecay>=0.4.12
```

Comment:
Is a good idea to pin the bugfix? Maybe just minor version it's ok, what do you think @lemointm ?

Reply:
I talked to Alex about this, we think that in the future the functionality TARDIS needs won't get deprecated

Reply:
but that shouldn't affect the bugfix number, right?

Reply:
I see what you're saying, sorry I misunderstood. The "bugfix" added some functionality, so it needs to be pinned... which is not the standard but that's how it ended up I suppose